lib/components/Information/Avatar/index.tsx

```diff
@@ -9,12 +9,13 @@ interface AvatarType {
   alt: string
   src?: string
   size?: ComponentProps<typeof AvatarComponent>["size"]
+  className?: ComponentProps<typeof AvatarComponent>["className"]
```

Comment:
I would really like to try and prevent people being able to pass `className` to components, as this will inevitably end with hacks. Can you think of any other solution to this involving the wrapper?

Reply:
Yeah, I was thinking that might be the case, I'll take a look.